data-collection/deploy/module-health-events.yaml

```
@@ -327,16 +330,17 @@ Resources:
               """
               logger.info(f"Event data: {event}")
               account = event.get('account')
-              batch_input = event.get('BatchInput')
-              items = event.get('Items')
-              if not (account or batch_input):
+              items = event.get('items')
+              if not (account):
                   raise ValueError(
                       "Please do not trigger this Lambda manually."
                       "Find the corresponding state machine in Step Functions and Trigger from there."
                   )
-              is_summary_mode = batch_input == None
+              is_summary_mode = not items
```

what if items is empty list? 

```suggestion
              is_summary_mode = items is None
```